static/src/javascripts/projects/commercial/modules/creatives/page-skin.js

```diff
@@ -40,19 +41,66 @@ const pageSkin = (): void => {
         }
     };
 
-    const initTopPositionOnce = (): void => {
+    const initTopPositionOnce = (hasTruskin: boolean): void => {
         if (topPosition === 0) {
             const navHeader = document.getElementsByClassName('new-header')[0];
             if (navHeader) {
-                topPosition = navHeader.offsetTop + navHeader.offsetHeight;
+                topPosition = hasTruskin
+                    ? navHeader.offsetTop + adLabelHeight
+                    : navHeader.offsetTop + navHeader.offsetHeight;
             }
         }
     };
 
-    // This is to reposition the Page Skin to start where the navigation header ends.
+    const shrinkElement = (element: HTMLElement | null): void => {
+        const frontContainer = document.querySelector('.fc-container__inner');
+        if (element && frontContainer) {
+            element.style.cssText = `max-width: ${
+                frontContainer.clientWidth
+            }px; margin-right: auto; margin-left: auto;`;
+        }
+    };
+
     const repositionSkin = (): void => {
-        if (hasPageSkin && isInAUEdition) {
-            initTopPositionOnce();
+        const bodyElement = document.querySelector('body');
+        const hasTruskin = bodyElement
+            ? bodyElement.classList.contains('truskin-page-skin')
```

Comment:
This class has been added into the new Truskin creative template https://github.com/guardian/commercial-templates/pull/227/files